src/main/webapp/resources/js/components/ncbi/export-table/NcbiExportTable.jsx

```diff
@@ -1,22 +1,32 @@
-import React, { useEffect, useState } from "react";
+import React from "react";
 import { Table } from "antd";
 import { getProjectNCBIExports } from "../../../apis/ncbi/ncbi";
 import { formatInternationalizedDateTime } from "../../../utilities/date-utilities";
 import { setBaseUrl } from "../../../utilities/url-utilities";
 import NcbiUploadStates from "../upload-states";
+import { getPaginationOptions } from "../../../utilities/antdesign-table-utilities";
 
 /**
  * Render a list of all Project NCBI Exports.
  * @returns {JSX.Element|string}
  * @constructor
  */
 export function NcbiExportTable() {
-  const [exports, setExports] = useState(null);
+  const [exports, setExports] = React.useState(null);
+  const [total, setTotal] = React.useState(0);
+  const [paginationOptions, setPaginationOptions] = React.useState(null);
 
-  useEffect(() => {
-    getProjectNCBIExports().then(setExports);
+  React.useEffect(() => {
+    getProjectNCBIExports().then((data) => {
+      setExports(data);
+      setTotal(data.length);
+    });
   }, []);
 
+  React.useMemo(() => {
```

Comment:
This should be:

```jsx
const paginationOptions = React.useMemo(() => {
  getPaginationOptions(total);
}, [total]);
```

Reply:
Updated in c96bd77d27718c9064e6535c7aac61dd11b3f80b